app/helpers/chain-actions.js

```diff
@@ -0,0 +1,25 @@
+import Ember from "ember";
+
+/**
+ * @module Helpers/chain-actions
+ * @description Allows running multiple successive actions
+ * @property {...functions} actions the actions to run
+ * @example
+ *
+ * <button {{action (chain-actions
+ *    (action "doThis")
+ *    (action "doThat")
+ *    (action "andThenThis")
+ *  )
+ * }}>
+ *  Submit
+ * </button>
+ *
+ */
+export default Ember.Helper.helper(function(actions) {
+  return async function() {
+    for (let action of actions) {
+      await action();
```

Comment:
Exceptions will be handled at `action` level ?

Reply:
Yeah, although you do have a point that it would carry on the chain regardless of errors. In which case it might be worth renaming to "multiple-actions" not to infer that it handles chain errors